packages/actor-scraper/playwright-scraper/src/internals/crawler_setup.ts

```diff
@@ -309,6 +310,16 @@ export class CrawlerSetup implements CrawlerSetupOptions {
         };
         const { context, state } = createContext(contextOptions);
 
+        if(this.input.closeCookieModals) {
```

Comment:
I am sure this could be caught by eslint, crawlee reports that as `keyword-spacing` error.

```suggestion
        if (this.input.closeCookieModals) {
```